src/Directory.Build.targets

```diff
@@ -30,6 +31,15 @@
     <RestoreProjectStyle>None</RestoreProjectStyle>
   </PropertyGroup>
 
+  <!-- Settings for GenerateReferenceAssemblySource. -->
+  <PropertyGroup>
+    <_LicenseHeaderTxtPath>$(RepositoryEngineeringDir)LicenseHeader.txt</_LicenseHeaderTxtPath>
+    <GenAPITargetPath>$([MSBuild]::NormalizePath('$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), Directory.Build.props))', 'ref', '$(TargetName).cs'))</GenAPITargetPath>
```

Comment:
Consider reusing the convention defined in https://github.com/dotnet/corefx/blob/master/eng/resolveContract.targets.  Not sure if it's worth it but you may want to move all these properties to a props fine under eng.

Reply:
I like the idea of having a props file for these, that way we can share it later if we wind up needing to set stuff for APICompat/GenFacades/etc